run_configs/gcom/gcom_rose_suite/suite.rc

```diff
@@ -0,0 +1,34 @@
+[scheduling]
+    [[dependencies]]
+        graph = """
+            grab_gcom => build_gcom_ar
+            grab_gcom => build_gcom_so
```

Comment:
Yes, we think that's a good idea when building an exe too because it also makes any linker error output much easier to navigate by removing the many .o files from the command in the error output.